### PR TITLE
Dev shravanth

### PR DESCRIPTION
**Pull request by shravanth1862**
create api routes, db methods. modified a few db methods to current requirements.

### CODE REVIEW

File: src/db.py [hunk 1]
```diff
@@ -241,14 +241,16 @@ def delete_list_by_id(list_id):
...
+            "INNER JOIN Tbl_Lists TL on Tbl_Elements.List_id = TL._id " \
+            "WHERE TL.Lists_Uuid = %s"
+
+    cursor.execute(query, (list_uuid,))
     retVal = cursor.fetchall()
-    my_db.commit()
+
     cursor.close()
     my_db.close()
     return retVal
```
Comment (SebTota) [inline src/db.py:256]: The return value in this scenario will be a list of tuples. There wont be any mappings to parse the data. Could you please instead turn this into a list of dictionary/json objects being returned.

**Review by SebTota — CHANGES_REQUESTED** [review]
Please make sure when you return all the items in a list that they are in a dictionary.

File: src/db.py (hunk 1 above)
Comment (shravanth1862) [reply src/db.py:256]: changed to return dictionary

**Comment by shravanth1862** [comment]
neet fix merge conflicts

**Pull request closed without merging**